stdlib/http/src/main/ballerina/src/http/websocket/websocket_client.bal

```diff
@@ -60,47 +59,51 @@ public type WebSocketClient client object {
     #
     # + data - Binary data to be sent
     # + finalFrame - Set to `true` if this is a final frame of a (long) message
-    # + return - `error` if an error occurs when sending
+    # + return - Returns a `WebSocketError` if an error occurs while sending the binary message to the server.
```

Comment:
This doesn't return WebSocketError even it states here. Btw why only this doesn't return WebSocketError?